### DISCUSSION TITLE
Anomalous space codes when copying from youtube-dl manpage

### DISCUSSION THREAD

**Discussion by Kratacoa** [kovidgoyal/kitty#3262] [Q&A]
As per title, when copying (using ctrl+shift+c) in the scrollback buffer the code snippet below and pasting it back into terminal, the result is that the line contains some form of space that is not interpreted as space by `bash` when executed, giving weird errors like "no such file or directory" for the option of the command.  
[Link to my configuration](https://github.com/Kratacoa/dotfiles/blob/master/config/kitty/.config/kitty/kitty.conf). Any clue as to why this happens?

```
  # Download YouTube playlist videos in separate direc  tory indexed by video order in a playlist
                $ youtube-dl -o '%(playlist)s/%(playlist_index)s - %  (title)s.%(ext)s' https://www.youtube.com/playlist?list=PLwiyx1dc3  P2JR9N8gQaQN_BCvlSlap7re
```

P.S. I copypasted it likewise from the terminal into the browser, and as you see there is a space before `(title)` that shouldn't be there, and the `list=..` part is separated whereas it shouldn't be, as can be seen by the manpage.

**Comment by kovidgoyal** [comment]
Not sure what you are asking. Can you provide a text file that when catted produces the problem text.

**Reply by Kratacoa** [reply]
I don't see how it's helpful, given that the problem was spotted in a manpage, but sure. [Here is the link](https://files.catbox.moe/0v2jni.md).
What I did was:
* Open the manpage
* Select the text with the touchpad
* Open the file with `kakoune`, enter insert mode and ctrl+shift+v to paste it.

I also did it with `xclip -o -sel clibpoard > test-kitty-copypaste1.md` and did a `diff` with the two files, and the only difference was the newline at the end, so it's not the editor.
You can see it if you search just the space in the newline, whether with `kakoune` or `vim`.

**Comment by kovidgoyal** [comment]
Because copying from the man page works fine for me, with less as the pager, so I need to see what exactly is happening for you. Are you using a custom pager for man?

**Reply by Kratacoa** [reply]
I'm using `less` as pager.
The only thing I could think of that's relevant to `less` is that I'm piping it through [source-highlight](https://www.gnu.org/software/src-highlite/), as can be seen in my [fish config](https://github.com/Kratacoa/dotfiles/blob/master/config/fish/.config/fish/config.fish) from line 29 to 38. I tried copying from `bash -c 'env -i man youtube-dl'` via scrollback buffer and the [result is even worse](https://files.catbox.moe/re7gln.md).
